spec/valued/mutable_spec.rb

```diff
@@ -1,149 +1,22 @@
+require 'spec_helper'
```

Comment:
You don't need to require `spec_helper`, it is automatically required by RSpec.

Reply:
`true`. Removed :+1: